packages/types/src/phase0/validator.ts

```diff
@@ -0,0 +1,67 @@
+import {ByteViews, ContainerNodeStructType, ValueOfFields} from "@chainsafe/ssz";
+import * as primitiveSsz from "../primitive/sszTypes.js";
+
+const {Boolean, Bytes32, UintNum64, BLSPubkey, EpochInf} = primitiveSsz;
+
+const NUMBER_2_POW_32 = 2 ** 32;
```

Comment:
We might consider having some utility packages for those kind of constants.

Reply:
this is so specific to this serialization, will refer to the source of this constant instead